src/App.js

```diff
@@ -1,5 +1,24 @@
-function App() {
-  return <div className="App"></div>;
+import { useState } from 'react';
+import styled from 'styled-components';
+import Wishlist from './Wishlist.js';
+import AddDiveWish from './AddDiveWish.js';
+
+export default function App() {
+  const [diveWishes, setDiveWishes] = useState([]);
+
+  return (
+    <>
+      <Heading>Diving Wishlist</Heading>
+      <AddDiveWish onAddDiveWish={AddToWishlist} />
+      <Wishlist diveWishes={diveWishes} />
+    </>
+  );
+
+  function AddToWishlist(destination, notes) {
```

Comment:
Only components start with a capital letter.
```suggestion
  function addToWishlist(destination, notes) {
```

Reply:
Done!